CRM/Case/Form/Activity/ChangeCaseStatus.php

```diff
@@ -145,7 +145,7 @@ public static function formRule($values, $files, $form) {
   public static function beginPostProcess(&$form, &$params) {
     $params['id'] = CRM_Utils_Array::value('case_id', $params);
 
-    if ($params['updateLinkedCases'] === '1') {
+    if (!empty($params['updateLinkedCases']) && ($params['updateLinkedCases'] === '1')) {
```

Comment:
Can be also written as

`if (CRM_Utils_Array::value('updateLinkedCases', $params) === '1')) {`

Reply:
Hi @pradpnayak I've updated it although I've always avoided that function for a few reasons. Maybe I will make a gitlab ticket about it.

Reply:
@pradpnayak Just for interest: https://lab.civicrm.org/dev/core/issues/937